### PR TITLE
Wimstockman patch 1

### PR DESCRIPTION
**Pull request by wimstockman**
Left The Treeview out

### CODE REVIEW

File: plugins/preview-tabbed [hunk 1]
```diff
@@ -138,7 +139,11 @@ previewer_loop () {
 
         kill_viewer
 
-        MIME="$(file -b --mime-type "$FILE")"
+       if which xdg-mime >/dev/null 2>&1 ; then
+          MIME="$(xdg-mime query filetype "$FILE")"
```
Comment (jarun) [inline plugins/preview-tabbed:143]: Alignment.

File: plugins/preview-tabbed [hunk 2]
```diff
@@ -169,8 +174,15 @@ previewer_loop () {
...
+            application/postscript)
+                if which zathura >/dev/null 2>&1 ; then
+                    zathura -e "$XID" "$FILE" &
+                else
+                    term_nuke "$XID" "$FILE"
+                fi
+                ;;
             inode/directory)
-                $TERMINAL "$XID" -e nnn "$FILE" &
+	        $TERMINAL "$XID" -e nnn "$FILE" &
```
Comment (jarun) [inline plugins/preview-tabbed:185]: Alignment.

File: plugins/preview-tabbed [hunk 3]
```diff
@@ -50,16 +50,17 @@
 
 
 XDOTOOL_TIMEOUT=2
-PAGER=${PAGER:-"vim -R"}
+PAGER=${PAGER:-"less -r"}
 NUKE="${XDG_CONFIG_HOME:-$HOME/.config}/nnn/plugins/nuke"
 
 
-if which xterm >/dev/null 2>&1 ; then
+
```
Comment (jarun) [inline plugins/preview-tabbed:57]: Please remove this newline and the blank line above.

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
It would have been good enough to update the old PR. I am closing this.

Also, please do not raise PRs on `nnn` if you do not understand the importance of alignment and cannot do it without redundant newlines. At this stage of the project, it's too difficult for us to waste time on these trivial things which should not have been there in the first place.

It's not too difficult to have your own personal preferences in some plugins as `nnn` asks to merge when you install a new version.

**Comment by wimstockman** [comment]
Hi, Jarun,
Thanks for your comment.
I updated the previous PR.
I'm sorry for the trouble. Will pay more attention in the future.
Keep up the good work.
Kind regards
Wim

**Pull request closed without merging**